src/spikeinterface/widgets/peaks_on_probe.py

```diff
@@ -0,0 +1,176 @@
+from __future__ import annotations
+
+import numpy as np
+
+
+from .base import BaseWidget, to_attr
+
+
+class PeaksOnProbeWidget(BaseWidget):
+    """
+    Generate a plot of spike peaks showing their location on a plot
+    of the probe. Color scaling represents spike amplitude.
+
+    The generated plot overlays the estimated position of a spike peak
+    (as a single point for each peak) onto a plot of the probe. The
+    dimensions of the plot are x axis: probe width, y axis: probe height.
```

Comment:
We may need to think about this. I usually hear probe width and probe depth. We should think about the terminology we are using so that we can be consistent across the codebase + probeinterface!

Reply:
Thanks @zm711! I have changed here to probe depth, do you know if we have an issue for such things? At some point I think we discussed it might be nice to add a glossary to the docs, maybe this kind of thing could go there?

Reply:
No we don't, but I really like the idea for a glossary. @h-mayorquin recently pointed me to the numpy glossary which is useful (I didn't realize it existed before). We should do that as part of our doc to-do list.